src/contributors.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+contributors_json = {}
+
+
+def get_contributors():
```

Comment:
We need two methods:
- one that updates the contributors_json variable based on the contents of the file
- one that returns the value of contributors_json

Rename this method to make it clearer that it's the first method. We also need to create the second method:

```suggestion
def get_contributors():
  return contributors_json

def update_contributors():
```